README.md

```diff
@@ -1 +1,55 @@
 # Conference-page
+
+This project is based on an online website for a Facebook instant games conference.
+
+![screenshot](./screenshots/tempsnip.png)
+![screenshot](./screenshots/mobile-screenshot.jpg)
+
+## Built With
+
+- Major languages : Html, css & Javascript
+- No Frameworks
+
+## Live Demo
+
+[Live Demo Link]()
```

Comment:
Great work using GitHub pages to deploy your web pages but your link is broken, it shows a 404 page so kindly fix that.